requirements.txt

```diff
@@ -6,3 +6,4 @@ pylint==2.13.9
 jsonschema==3.2.0
 json-schema-for-humans==0.39.1
 openpyxl==3.0.9
+rfc3339-validator==0.1.4
```

Comment:
If we've removed the `format: date` from the dates in the spec, I don't think we need this.

Reply:
Actually, I added this dependency because [python-jsonschema](https://python-jsonschema.readthedocs.io/en/stable/validate/?highlight=format#jsonschema.FormatChecker) said it was needed to support `format: datetime`.  But if we do `date`, it's built in